test/e2e/tsdx-build-default.test.js

```diff
@@ -31,6 +31,15 @@ describe('tsdx build :: zero-config defaults', () => {
     expect(output.code).toBe(0);
   });
 
+  it("shouldn't compile files in test/ or types/", () => {
+    const output = execWithCache('node ../dist/index.js build');
+
+    expect(shell.test('-f', 'dist/test/')).toBeFalsy();
+    expect(shell.test('-f', 'dist/types/')).toBeFalsy();
```

Comment:
This should be [`-d` or `-e`](https://github.com/shelljs/shelljs#testexpression), not `-f`, which specifically checks for a file. I could test for the specific files too, but would prefer to keep it generic, as the directories shouldn't exist at all.

Reply:
used `-d` and tested to make sure it does give a test error when `test/` is erroneously in `include`.